doc/api/tty.md

```diff
@@ -88,6 +88,17 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* On Windows resize events will be emitted only if stdin is unpaused 
+(by a call to resume() or by adding data listener) and in raw mode. It can be 
```

Comment:
`resume()` -> `\`resume()\``

`adding data` -> `adding a data`

`can be also` -> `can also be`